src/commcare_cloud/ansible/roles/couchdb2/templates/restore_couchdb_backup.sh.j2

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+BACKUP=$1
+TODAY=$(date +"%Y_%m_%d")
+
+read -p 'Are you sure you want to restore couchdb to a past state [y/N]: ' -r confirm
+if [ $confirm != 'y' ]; then
+    exit 1
+fi
+read -p 'This script will only work on single-node clusters. Damage may occur if this couchdb cluster has more than one node. Are you restoring to a single node? [y/N]: ' -r confirm_nodes
+if [ $confirm_nodes != 'y' ]; then
+    exit 1
+fi
+read -p "This will delete the current db and replace it with the contents of ${BACKUP}. Are you SURE you are sure [y/N]: " -r confirm2
```

Comment:
Not sure how best to word this, but `current db` is a little confusing since its talking about the entirety of the CouchDB instance and the namespaces in couch are also called `db`. 

I don't see an instance in this script that calls `commcare__users` or any of the dbs `db` though, which is good.